parser/ParseSubqueryExpression.cpp

```diff
@@ -27,7 +29,8 @@ namespace quickstep {
 class ParseTreeNode;
 
 std::string ParseSubqueryExpression::generateName() const {
-  LOG(FATAL) << "ParseSubqueryExpression::generateName() is not implemented yet";
+  // TODO(jianqiao): generate a more informative name for ParseSubqueryExpression.
```

Comment:
According to the code style, we capitalize the first letter of the first word in a sentence. In this case, `Generate`.